flutter_appauth/example/lib/main.dart

```diff
@@ -210,6 +210,9 @@ class _MyAppState extends State<MyApp> {
     try {
       _setBusyState();
       // use the discovery endpoint to find the configuration
+      // the code_challenge is generated from the code verifier and only the code verifier is included in the result
```

Comment:
A few things
- as the comment that was there in the previous line before is no longer the only comment for code below, could you switch to a using [multi-line comments](https://dart.dev/language/comments#multi-line-comments)?
- are you able to add a full stop to break up the various sentences?
- are you able to update the comment you've added so it's consistent with how it expresses terms from RFC? There's a mix with how words are expressed (i.e. code_challenge, code verifier vs codeVerifier and authorizationCode). My suggestion would be to express this more like plain English, that is use a space to separate. In this case it would be to use "code challenge", "code verifier" and "authorisation code" (or "authorization code" if you're going by American English)